---
layout: default
title: Research
subtitle: uncovering genetic drivers of rare human diseases
---

> I develop methods to analyze genome sequencing data in the context of other 'omics and clinical 
> health data to prioritize and functionally interpret genetic variants with roles in human disease. See
> [my CV](../assets/ShilpaKobren_CV.pdf) for details.

![integration of patient phenotype in variant prioritization](../assets/img/clinical_integration.png)
*Image from our [Nat Comm, 2023](https://www.nature.com/articles/s41467-023-41980-6) paper.*

### Functional genomics for "N of 1" analyses

The genome is a **big space**, and accurately pinpointing variants that underlie specific human health conditions is a formidable challenge.
Traditionally, genes have been treated as black box functional units, but we now know that individual variants within and between genes 
can have wildly different impacts. Because comprehensive, *in vivo* (in a living system) functional assessment of all possible genetic variants is (currently) infeasible, 
we instead turn to *in silico* (computational) variant functionality predictions. We develop integrative tools for assessing
the functionality of specific genomic positions and are interested in leveraging multimodal biological and biomedical data to derive new insights on the 
function of genetic variants. \[[30535108](https://pubmed.ncbi.nlm.nih.gov/30535108/), [33580225](https://pubmed.ncbi.nlm.nih.gov/33580225/)\]

### Integration of clinical phenotyping 

Patient clinical phenotyping data is an essential component in interpreting the impact of genetic variants on human health.
Phenotyping data can be noisy, unstructured, and difficult to obtain, and utilizing this information often requires deep clinical intuition. 
We are interested in developing computational approaches for streamlining the process of utilizing (standardized) phenotype data 
for automating diagnostic gene prioritization and interpretation. \[[37828001](https://pubmed.ncbi.nlm.nih.gov/37828001/), [medRxiv](https://www.medrxiv.org/content/10.1101/2022.12.07.22283238v1), [bioRxiv](https://www.biorxiv.org/content/10.1101/2024.02.13.580158v1)\]



### Deriving insights from population-level analyses

Even though the genome is a big space, it is also a **finite space**. This means that as the number of sequenced genomes continues to grow,
we will begin to observe all possible variants (and recurrence of functional variants in phenotypically-matched cohorts). Indeed, the number of sequenced 
tumor genomes has surpassed 10s of thousands, collective cohorts of sequenced Mendelian patients is exceeding 100s of thousands, and sequenced
diverse, healthy populations is set to pass a million or more. By integrating variant functionality information, evolutionary constraint and mutational models,
we will have the power to detect extremely rare variants that play roles in human cancers and other diseases. \[[32711844](https://pubmed.ncbi.nlm.nih.gov/32711844/), [bioRxiv](https://www.biorxiv.org/content/10.1101/2024.02.13.580158v1)\]

### Publications
:star: = *project lead*, :love_letter: = *corresponding author*, :busts_in_silhouette: = *team science*

* Joint, multifaceted genomic analysis enables diagnosis of diverse, ultra-rare monogenic presentations. [*bioRxiv*](https://www.biorxiv.org/content/10.1101/2024.02.13.580158v1), 2024. :star:
* Simulation of undiagnosed patients with novel genetic conditions. [*Nature Communications*](https://www.nature.com/articles/s41467-023-41980-6), 2023. :love_letter:
* Commonalities across computational workflows for uncovering explanatory variants in undiagnosed cases. [*Genetics in Medicine*](https://www.nature.com/articles/s41436-020-01084-8), 2021. :star:
* Innovative methodological approaches for data integration to derive patterns across diverse, large-scale biomedical datasets. [*Pac Symp Biocomput*](https://psb.stanford.edu/psb-online/proceedings/psb21/intro-pattern_rec.pdf), 2021. :love_letter:
* How medical mysteries push back the frontier of genomics knowledge. [*UDN PEER Newsletter*](https://mailchi.mp/659a8ae4c89a/udn-peer-newsletter-undiagnosed-diseases-network-summer-2021), 2021. :star:
* PertInInt: An integrative, analytical approach to rapidly uncover cancer driver genes with perturbed interactions and functionalities. [*Cell Systems*](https://www.cell.com/cell-systems/fulltext/S2405-4712(20)30202-7), 2020. :star:
* Ongoing challenges and innovative approaches for recognizing patterns across large-scale, integrative biomedical datasets. [*Pac Symp Biocomput*](https://psb.stanford.edu/psb-online/proceedings/psb20/Intro_Biomed%20Data.pdf), 2020. :star:
* Systematic, domain-based aggregation of protein structures highlights DNA-, RNA-, and other ligand-binding positions. [*Nucleic Acids Research*](https://academic.oup.com/nar/article/47/2/582/5232439), 2019. :star:
* Pervasive variation of transcription factor orthologs contributes to regulatory network divergence. [*PLoS Genetics*](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005011), 2015. :star:
* Formatt: Correcting protein structural alignments by sequence peeking. [*ACM-BCB'11*](https://dl.acm.org/citation.cfm?doid=2147805.2147842), 2011. :star:
* Phenotypic overlap between rare disease patients and variant carriers in a large population cohort informs biological mechanisms. [*medRxiv*](https://www.medrxiv.org/content/10.1101/2024.04.18.24305861v1), 2024. :love_letter:
* VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders. [*medRxiv*](https://www.medrxiv.org/content/10.1101/2024.04.15.24305876), 2024. :love_letter:
* Deep learning for diagnosing patients with rare genetics diseases. [*medRxiv*](https://www.medrxiv.org/content/10.1101/2022.12.07.22283238v1), 2023. :busts_in_silhouette:
* Polygenic risk scores for autoimmune related diseases are significantly different and skewed in cancer exceptional responders. [*medRxiv*](https://www.medrxiv.org/content/10.1101/2023.02.22.23285773v2), 2023. :busts_in_silhouette:
* RExPRT: a machine learning tool to predict pathogenicity of tandem repeat loci. [*Genome Biology*](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03171-4, 2024.), 2024. :busts_in_silhouette:
* The contribution of mosaicism to genetic diseases and de novo pathogenic variants. [*Am J Med Genet Part A*](https://onlinelibrary.wiley.com/doi/abs/10.1002/ajmg.a.63309), 2023. :busts_in_silhouette:
* Formatt: Correcting protein structural alignments by incorporating sequence alignment. [*BMC Bioinformatics*](https://www.biomedcentral.com/1471-2105/13/259), 2012. :busts_in_silhouette:
* Evolving soft robotic locomotion in PhysX. [*ACM-GECCO'09*](https://dl.acm.org/citation.cfm?doid=1570256.1570351), 2009. :busts_in_silhouette:
* A patient-centric information commons for a national undiagnosed diseases network. *In preparation*, 2024. :busts_in_silhouette: